packages/react-client/src/index.ts

```diff
@@ -47,7 +49,7 @@ export type {
   VadStatus,
   EncodingReason,
   MetadataParser,
-  ConnectConfig,
+  ConnectConfig as TSClientConnectConfig,
```

Comment:
Do we want to export `TSClientConnectConfig` here, or version modified in one of flies below?

Reply:
The new one. Fixed